python/snewpy/models/base.py

```diff
@@ -161,6 +162,33 @@ def get_transformed_spectra(self, t, E, flavor_xform):
 
         return transformed_spectra   
 
+    def get_flux (self, t, E, distance, flavor_xform=NoTransformation(), **kwargs):
```

Comment:
```suggestion
    def get_flux (self, t, E, distance, flavor_xform=NoTransformation()):
```
If the `**kwargs` aren’t used (or even passed through to some other function), I would delete them here.

Reply:
You're right. What I had in mind is to be compatible with all the models, including Fornax etc. But this is a separate issue #223 
For this PR I should remove this.